src/site/_includes/partials/script-loader-embed.njk

```diff
@@ -0,0 +1,30 @@
+{# Sourced scripts cannot be authorized by a CSP hash individually, so we need
+to use an inline script loader for any sourced scripts. #}
+{# This works because the CSP includes the 'strict-dynamic' keyword, allowing
+the children of an authorized script to run. #}
+{% set scriptLoader %}
+function loadScript(url, type) {
```

Comment:
Is this different from https://github.com/GoogleChrome/web.dev/blob/85422a4f1c431f4ca2ce60a7e8f9411f5d5e0e5a/src/site/_includes/partials/script-loader.njk#L12 ?

Reply:
I've rolled our own stripped down version from the Chrometober repo instead.